### DISCUSSION TITLE
To launch Exo I had to downgrad elastic search  to 2.0.4

### DISCUSSION THREAD

**Discussion by mgiammarco** [exoplatform/exo-community#62] [Q&A]
Hi,
I am trying exo using docker-compose.
I have downloaded the script from link suggested in guide and it does not start because elastic search 2.1.0 gives this error:
`Transport SSL must be enabled if security is enabled. Please set [xpack.security.transport.ssl.enabled] to [true] or disable security by setting [xpack.security.enabled] to [false]`

I have downgraded to 2.0.4 and it seems to work.
I see that Exo version is 6.5, is it a stable version?
Thanks,
Mario

**Comment by rdenarie** [comment]
Hello,
You're, the environment variable `xpack.security.enabled=false` is missing for ES component.
We just update the docker compose file, in order to add it.
Can you update your file an retry ?

eXo Platform 6.5 is the last stable version, just released few days ago.
